<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timetable</class>
 <widget class="QDialog" name="Timetable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>991</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="Messanger_btn">
        <property name="text">
         <string>메신저 열기</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="locale">
         <locale language="Korean" country="SouthKorea"/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="WeekSCheduleAdd_btn">
          <property name="text">
           <string>주간 일정 추가</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="WeekSCheduleDelete_btn">
          <property name="text">
           <string>주간 일정 삭제</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QDateTimeEdit" name="CurrentTime">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::PreventContextMenu</enum>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="currentSectionIndex">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>이날의 주간 일정</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="WeeklySchedule_list">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>이날의 그룹 일정</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="GroupSchedule_list"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>이날의 일일 일정</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="DailySchedule_list"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="DaySCheduleAdd_btn">
          <property name="text">
           <string>일일 일정 추가</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DaySCheduleDelete_btn">
          <property name="text">
           <string>일일 일정 삭제</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
